.test-infra/kubernetes/hadoop/SmallITCluster/setup-all.sh

```diff
@@ -0,0 +1,37 @@
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+#!/bin/sh
+set -e
+
+kubectl create -f hdfs-single-datanode-cluster.yml
```

Comment:
Are the shell scripts for manually setting up a cluster ? PerfKit-based executions already perform starting up and shutting down of the cluster, don't they ?

Reply:
Yes those scripts are for manual spinning of the cluster. PerfKit-based executions require to provides kubernetes .yml files and PerfKit takes care or spinning up of the infrastructure and passing variables to pipeline options. 

In our case we will use LoadBalancer IP to modify filenamePrefix and hdfsConfiguration variables. I will provide a link with an example, but my initial goal was to submit that as separate PR.